CMakeLists.txt

```diff
@@ -108,3 +130,18 @@ appx_add_test(TestXMLEscaping)
 appx_add_test(TestZIPEscaping)
 appx_add_test(TestContentTypes)
 appx_add_test(TestEmptyFile)
+
+MESSAGE(STATUS "--------------------------------------------------------------------------------")
```

Comment:
I'm not a big fan of status outputs like this during configuration. They're helpful for debugging but too much information to have every single configure.